deltachat-ios/Controller/ChatListViewController.swift

```diff
@@ -1086,14 +1087,13 @@ extension ChatListViewController: ChatListEditingBarDelegate {
                 }
             })
         } else {
-            alert.addAction(UIAlertAction(title: String.localized("menu_unmute"), style: .default) { [weak self] _ in
+            actions.append(UIAction(title: String.localized("menu_unmute"), image: UIImage(systemName: "speaker.wave.2")) { [weak self] _ in
```

Comment:
We do the `guard let self else { return }` dance on several places, but we don't use that unwrapped `self`, it seems to me. What am I missing?

-----

From what I understood, the dance is to break a retain cycle, but this way we still use the old, strong references?

Reply:
You can use implicit self after guard let self since swift 5.8: https://github.com/swiftlang/swift-evolution/blob/main/proposals/0365-implicit-self-weak-capture.md

Reply:
til

Reply:
k, this was also my impression as someone not that deep in swift: that `self` can now be omitted in more cases, and that this seems to be best practise

Reply:
We're using Swift 5.5 (it seems)

Reply:
Hmm, really? The CI is giving Swift 6 errors